cmake/modules/boards.cmake

```diff
@@ -131,7 +131,7 @@ foreach(root ${BOARD_ROOT})
     message(WARNING "BOARD_ROOT element without a 'boards' subdirectory:
 ${root}
 Hints:
-  - if your board directory is '/foo/bar/boards/<ARCH>/my_board' then add '/foo/bar' to BOARD_ROOT, not the entire board directory
+  - if your board directory is '/foo/bar/boards/plank.dts' then add '/foo/bar' to BOARD_ROOT, not the entire board directory
```

Comment:
not sure where the .dts comes from? This should be `/foo/bar/boards/<VENDOR>/my_board`

Reply:
OK, but it does not say where the dts files is supposed to be in the tree, should I try to add more clarifications on this? Like "Your dts file has be placed in a directory named ’boards’?

Reply:
this is a board directory as the comment clearly states, it has nothing to do with a dts file